test/metabase/api/metric_test.clj

```diff
@@ -416,7 +416,7 @@
                {:name                   "Metric B"
                 :id                     id-2
                 :creator                {}
-                :definition_description "Venues, Sum of Category → Name, Filtered by Price is equal to 4 and Segment"}]
+                :definition_description "Venues, Sum of Category → ID, Filtered by Price is equal to 4 and Segment"}]
               (filter (fn [{metric-id :id}]
                         (contains? #{id-1 id-2 id-3} metric-id))
                       (mt/user-http-request :rasta :get 200 "metric/")))))))
```

Comment:
These changes fix a nonsense aggregation, they are not strictly necessary for this PR.